src/commands/synthetics/utils.ts

```diff
@@ -203,7 +203,8 @@ export const getResultOutcome = (result: Result): ResultOutcome => {
 }
 
 export const getSuites = async (GLOB: string, reporter: MainReporter): Promise<Suite[]> => {
-  reporter.log(`Finding files in ${path.join(process.cwd(), GLOB)}\n`)
+  reporter.log(`Finding files matching ${getGlobForDisplay(GLOB)}\n`)
```

Comment:
For simplicity, what do you think about using `resolve`?
https://nodejs.org/api/path.html#pathresolvepaths

```suggestion
  reporter.log(`Finding files matching ${path.resolve(process.cwd(), GLOB)}\n`)
```

Reply:
Thank you, I was so convinced that a glob like `{,!(node_modules)/**/}*.synthetics.json` would not be supported that I didn't try it.

Indeed, the following works:
```
> require('path').resolve('.', '{,!(node_modules)/**/}*.synthetics.json')
'/Users/corentin.girard/go/src/github.com/DataDog/synthetics-worker.git/prod/{,!(node_modules)/**/}*.synthetics.json'
> require('path').resolve('.', '/example/*.synthetics.json')
'/example/*.synthetics.json'
```